src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/database/jpa/entity/induction/PreviousQualificationsEntity.kt

```diff
@@ -36,8 +36,6 @@ import java.util.UUID
  * Holds details about a Prisoner's educational qualifications, including where relevant, the grades achieved in each
  * subject.
  *
- * Note that the list of `qualifications` can be empty, but `educationLevel` is mandatory (but only if the Prisoner has
- * been asked about their education).
```

Comment:
This comment is no longer true/relevant (since we did the question set changes a few weeks back) 
(and to be fair that comment should not really have been related to the database layer; it was more relevant at the API and possibly domain layers)